### PR TITLE
Sqlite backend 2.5

### PR DESCRIPTION
**Pull request by llybin**
if ok I will create PR with migration_sqlite/V1__initial.sql on develop branch
I don't add sqlite vacuum because I think need get rid that for all db backends. Optimizations db locks table, it's heavy operation which should to do admin manually. For mysql, sqlite optimization tables works through creating table from a scratch and copying datas in it. h2 don't know.

### CODE REVIEW

File: conf/brs-default.properties [hunk 1]
```diff
@@ -211,7 +211,7 @@ API.SSL_keyStorePassword = password
 # DB.Password=
 
 # We are using file based as default
-DB.Url=jdbc:h2:file:./burst_db/burst;DB_CLOSE_ON_EXIT=FALSE
+DB.Url=jdbc:sqlite:./burst_db/burst.sqlite
```
Comment (jjos2372) [inline conf/brs-default.properties:214]: I think this needs to be experimental for a while, better make it the default only after some heavy testing. Please leave both h2 and sqlite with the sqlite version commented out.

**Comment by llybin** [comment]
Currently I syncing testnet, I let you know later. If everything good, next, need fully sync mainnet, then I will fully sure sqlite is stable. I will ask your help how to sync mainnet with that version.

**Comment by jjos2372** [comment]
> Currently I syncing testnet, I let you know later. If everything good, next, need fully sync mainnet, then I will fully sure sqlite is stable. I will ask your help how to sync mainnet with that version.

Great, sure I'll give you the details to sync with mainnet.

**Comment by llybin** [comment]
image

**Comment by llybin** [comment]
mainnet synced fully. Faced one problem(2-3 times), which doesn't cause freezes. Records were updated and inserted, checked. With SQLITE needed work better with one connection. So if make this changes should be all ok. Next, can try disable WAL and compare diffrences.

image
image
image

**Comment by llybin** [comment]
mainnet synced fully. Faced one problem(2-3 times), which doesn't cause freezes. Records were updated and inserted, checked. With SQLITE needed work better with one connection. So if make this changes should be all ok. Next, can try disable WAL and compare diffrences.

image
image
image

**Comment by llybin** [comment]
https://github.com/llybin/burstcoin/commits/sqlite_backend

**Comment by jjos2372** [comment]
Thanks, I made a manual merge (due to conflicts) on this branch https://github.com/burst-apps-team/burstcoin/tree/jjos/sqlite2

**Comment by llybin** [comment]
added commit error gone

**Pull request closed without merging**